packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2061,6 +2094,20 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     return SelectionResult.end;
   }
 
+  /// Selects a word in a selectable at the location
+  /// [SelectWordSelectionEvent.globalPosition].
+  @protected
+  SelectionResult handleSelectWord(SelectWordSelectionEvent event) {
+    return _handleSelectTextBoundary(event);
+  }
+
+  /// Selects a paragraph in a selectable at the location
+  /// [SelectParagraphSelectionEvent.globalPosition].
+  @protected
+  SelectionResult handleSelectParagraph(SelectParagraphSelectionEvent event) {
```

Comment:
Is it worth it to deprecate `handleSelectWord` and expose only `handleSelectTextBoundary`?  Maybe that's a question for @chunhtai.

Reply:
yes I think `handleSelectTextBoundary` is more scalable, but we need a better name. The content may not be texts, so `TextBoundary` may be too opinionated